File: Python Projects/Lógica de Programação  I/Exercícios/Lista9/Exercicio1.py
MAX = 100
nome = list(range(MAX))
contador  = 0

while contador < MAX:
    nome[contador] = str (input("Digite o nome da {} pessoa: ".format(contador+1)))
    contador += 1
contador = 0
print("\nlista de nomes: ")
while contador < MAX:
    print (nome[contador])
    contador += 1

File: Java/Lógica/Lista 9/src/Exercicio1.java

import java.util.Scanner;

public class Exercicio1 {
	private static Scanner leia;

	public static void main(String args[]) {
		leia = new Scanner(System.in);

		final int MAX = 100;
		int contador = 0;
		String[] nome = new String[MAX];

		System.out.println("\nLer 100 números de matrículas de alunos e armazenar em um vetor");

		while (contador < MAX) {
			System.out.println("\nDigite o nome da " + (contador + 1) + "º pessoa: ");
			nome[contador] = leia.next();
			contador++;
		}
		contador = 0;
		System.out.println("\n\tLISTA DE NOMES\n       ----------------\n");

		while (contador < MAX) {
			System.out.println("\t → " + nome[contador]);
			contador++;
		}

	}
}
